lib/crypto/c_src/dh.c

```diff
@@ -270,7 +270,7 @@ ERL_NIF_TERM dh_generate_key_nif(ErlNifEnv* env, int argc, const ERL_NIF_TERM ar
     BIGNUM *dh_p_shared;
     BIGNUM *dh_g = NULL;
     BIGNUM *priv_key_in = NULL;
-    unsigned long len = 0;
+    ErlNifUInt64 len = 0;
```

Comment:
This `len` can stay `unsigned long` I think. It's never used as a 64-bit integer.

Reply:
You're right.